src/new_storage/storage_manager/handle.rs

```diff
@@ -98,7 +90,7 @@ enum Task {
 
 /// The `WriteTask` struct represents a write task
 #[derive(Debug)]
-struct WriteTask {
+pub struct WriteTask {
     /// The cache manager.
     cache: Arc<Mutex<MemoryCache<CacheKey, LruPolicy<CacheKey>>>>,
```

Comment:
"cache, block, ino" is duplicated with FileHandleInner,  should be removed

Reply:
Ok, delete `ino` and change the function to FIleHandleInner.